torch/distributed/tensor/parallel/fsdp.py

```diff
@@ -242,7 +242,9 @@ def _chunk_dtensor(
     # We need to explicitly call .detach() to return a new tensor detached from the current graph.
     tensor = tensor.clone().detach()
 
-    # if a tensor has not yet sharded by TP
+    # When a layer is not involved in TP, then the tensor will not be a DTensor.
+    # e.g. When a layer is not sppecified in the parallelize_plan, TP will have no effect on the layer.
+    # e.g. When you do PairwiseParallel on a 3 layer model, TP will have no effect on the third layer.
```

Comment:
Maybe we can update the docs and tests later: we don't want to promote PairwiseParallel as it's not easy to use, going forward we are promoting ColwiseParallel and RowwiseParallel instead.